llvm/test/Transforms/SimplifyCFG/X86/switch_to_lookup_table.ll

```diff
@@ -2039,3 +2039,32 @@ return:
   %x = phi i8 [ 3, %sw.default ], [ 124, %sw.bb3 ], [ -99, %sw.bb2 ], [ -66, %sw.bb1 ], [ -33, %entry ]
   ret i8 %x
 }
+
+define i8 @linearmap_dec_wrapped_mon(i3 %0) {                              ; Assume 1
```

Comment:
Not sure if you really want to leave those "; Assume 1" comments etc that I made in the testcase when you submit it?